src/NorESM/cam_diagnostics.F90

```diff
@@ -1134,7 +1146,7 @@ subroutine diag_init(pbuf2d)
          history_eddy_out   = history_eddy    , &
          history_budget_out = history_budget  , &
          history_budget_histfile_num_out = history_budget_histfile_num, &
-         history_waccm_out  = history_waccm)
+         history_waccm_out  = history_waccm)       
```

Comment:
Please always remove whitespace at the end of lines. Your editor should be able to do this automatically whenever saving a file.